packages/gallery/example/src/index.html

```diff
@@ -0,0 +1,401 @@
+                            <picture><source srcset="https://placehold.co/1630x1500.jpg" media="(min-width:1300px)"><source srcset="https://placehold.co/1300x1200.png" media="(min-width:730px)"><source srcset="https://placehold.co/730x300.png" media="(min-width:430px)"><img loading="lazy" src="https://placehold.co/430x300.png" alt="Test image"></picture>
+                        </div>
+                        <div class="gallery__item-footer">
+                            <div class="gallery__item-meta">
+                                <div class="gallery__item-description">
+                                    <h2 class="gallery__item-artist">Kate Murphy</h2>
+                                    <div class="gallery__item-category">Architecture - May</div>
+                                    <div class="gallery__item-title">Example project name here</div>
+                                </div>
+                                <!-- <ul class="gallery__item-tags">
+                                    <li class="gallery__item-tag">Example tag</li>
+                                    <li class="gallery__item-tag">Example tag</li>
+                                </ul> -->
+                            </div>
+                        </div>
+                    </li>
+                    <li class="gallery__item" data-gallery-item>
+                        <div class="gallery__item-img-container">
+                            <!-- <div data-gallery-item-loader>Loading...</div> -->
+                            <picture><source srcset="https://placehold.co/1640x2400.jpg" media="(min-width:1400px)"><source srcset="https://placehold.co/1400x2200.png" media="(min-width:740px)"><source srcset="https://placehold.co/740x1200.png" media="(min-width:440px)"><img loading="lazy" src="https://placehold.co/430x300.png" alt="Test image"></picture>
+                        </div>
+                        <div class="gallery__item-footer">
+                            <div class="gallery__item-meta">
+                                <div class="gallery__item-description">
+                                    <h2 class="gallery__item-artist">Kate Murphy</h2>
+                                    <div class="gallery__item-category">Architecture - June</div>
+                                    <div class="gallery__item-title">Example project name here</div>
+                                </div>
+                                <!-- <ul class="gallery__item-tags">
+                                    <li class="gallery__item-tag">Example tag</li>
+                                    <li class="gallery__item-tag">Example tag</li>
+                                </ul> -->
+                            </div>
+                        </div>
+                    </li>
+
+
+                </ul>
+                
+                <button class="gallery__next" aria-label="Next image" data-gallery-next>
```

Comment:
When I was trying out examples on iOS Voiceover, it felt a bit better having the Next/Previous buttons as siblings in the markup.  When they were either side, it felt like I was "accidentally" scrolling the gallery and getting to the end of it, when I was just trying to reach the 'Next' button.

This is just my opinion though of course!! And comes with all the usual caveats around me not being an AT power user and just going by what feels right to me personally!  

Also, I'm very aware that this is just the example.  The example for this component feels quite important though, compared to the others - since a lot of the functionality is tied up in really specific markup and css.